Zend/zend_compile.h

```diff
@@ -384,6 +384,7 @@ typedef struct _zend_internal_arg_info {
 	const char *name;
 	zend_type type;
 	zend_uchar pass_by_reference;
+	const char *default_value;
```

Comment:
Should be after `is_variadic` or before `pass_by_reference`, otherwise you'll waste a lot of memory on padding.